hist/hist/inc/THnSparse.h

```diff
@@ -51,6 +51,13 @@ class THnSparse: public THnBase {
    THnSparse(const char* name, const char* title, Int_t dim,
              const Int_t* nbins, const Double_t* xmin, const Double_t* xmax,
              Int_t chunksize);
+   THnSparse(const char* name, const char* title,
+             const std::vector<TAxis*>& axes,
```

Comment:
I see the usecase for the `std::vector<TAxis> const&` constructor for the nested initializer lists. But what is the usecase for this constructor?